rclpy/test/test_timer.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2016 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import multiprocessing
+import sys
+import traceback
+
+
+def run_catch_report_raise(func, *args, **kwargs):
+    try:
+        return func(*args, **kwargs)
+    except:
+        print('exception in {0}():'.format(func.__name__), file=sys.stderr)
+        traceback.print_exc()
+        raise
+
+
+def func_walltimer(args):
+    import time
+    import functools
+    period = float(args[0])
+    max_time = float(args[1])
+    print('period: %f' % period)
+
+    def timer_cb(timestamps):
+        timestamps.append(time.time())
+
+    import rclpy
+
+    rclpy.init()
+    node = rclpy.create_node('talker')
+    timestamps = []
+    nb_iterations = 0
+
+    test_callback = functools.partial(timer_cb, timestamps=timestamps)
```

Comment:
done in https://github.com/ros2/rclpy/pull/54/commits/57550a0e4ce3b691b8c9645fb3ab8a06bf0ee11f